autologout.module

```diff
@@ -64,8 +70,9 @@ function autologout_timeout_validate($timeout, $max_timeout = NULL) {
 /**
  * Adds a field to user/edit to change that users logout.
  */
-function autologout_form_user_form_alter(&$form, FormStateInterface $form_state) {
-  $user = \Drupal::currentUser();
+function autologout_form_user_form_alter(&$form, $form_state) {
```

Comment:
hook_form_alter takes in FormStateInterface as a second argument
https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Form%21form.api.php/function/hook_form_alter/8
Why was this removed?